pyrasterframes/src/main/python/setup.py

```diff
@@ -47,6 +47,16 @@ def _divided(msg):
     return divider + '\n' + msg + '\n' + divider
 
 
+from pweave import PwebPandocFormatter
+
+class PegdownMarkdownFormatter(PwebPandocFormatter):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+    # Pegdown doesn't support the width and label options.
+    def make_figure_string(self, figname, width, label, caption=""):
+        return "![%s](%s)" % (caption, figname)
+
```

Comment:
This gets rid of a lot of figure insertion roaches due to non-standard Markdown.